static/app/views/performance/transactionSummary/relatedIssues.tsx

```diff
@@ -55,9 +55,11 @@ class RelatedIssues extends Component<Props> {
     });
     currentFilter.addQuery('is:unresolved').setTagValues('transaction', [transaction]);
 
-    // Filter out key_transaction from being passed to issues as it will cause an error.
-    currentFilter.removeTag('key_transaction');
-    currentFilter.removeTag('team_key_transaction');
+    // Filter out these search terms from being passed to issues as they will cause an error.
+    currentFilter
+      .removeTag('key_transaction')
+      .removeTag('team_key_transaction')
```

Comment:
should we add these two to `TRACING_FIELDS`?

Reply:
Good call, I've moved them into `TRACING_FIELDS`. They **will not** appear as an option in discover

Reply:
🙏 thanks for checking that